aws_marketplace/curating_aws_marketplace_listing_and_sample_notebook/Algorithm/curating_good_algorithm_listing.md

```diff
@@ -0,0 +1,46 @@
+It is important that you make it easy for users with different levels of experience to use your algorithm in AWS Marketplace. To do this, we recommend that you include the following elements as part of your AWS Marketplace algorithm listing. 
```

Comment:
maybe phrase like this?

```suggestion
It is important that algorithms in the AWS Marketplace are easily accessible to users with different levels of experience. To do this, we recommend the following elements be included as part of your AWS Marketplace algorithm listing. 
```

Reply:
I would like to retain the word "use" than saying accessible. Algorithms are accessible via portal.